packages/shorebird_cli/lib/src/commands/patch/ios_patcher.dart

```diff
@@ -220,8 +188,9 @@ https://docs.shorebird.dev/status#link-percentage-ios
         releaseArtifact: releaseArtifactFile,
       );
       if (exitCode != ExitCode.success.code) return exit(exitCode);
-      if (linkPercentage != null && linkPercentage < minLinkPercentage) {
-        logger.warn(lowLinkPercentageWarning(linkPercentage));
+      if (linkPercentage != null &&
+          linkPercentage < Patcher.minLinkPercentage) {
```

Comment:
This could also be `if ((linkPercentage ?? 0) < Patcher.minLinkPercentage)`

Reply:
Yup good call will update

Reply:
Ah, fair

Reply:
I can add a helper to `Patcher` like `shouldShowLinkPercentageWarning`

Reply:
This is totally fine as-is, I was just suggesting a super minor bit of shorthand